src/Agent.Sdk/Knob/AgentKnobs.cs

```diff
@@ -19,6 +19,13 @@ public class AgentKnobs
             new RuntimeKnobSource("VSTS_SETUP_DOCKERGROUP"),
             new EnvironmentKnobSource("VSTS_SETUP_DOCKERGROUP"),
             new BuiltInDefaultKnobSource("true"));
+        
+        public static readonly Knob AllowMountTasksReadonlyOnWindows = new Knob(
+            nameof(AllowMountTasksReadonlyOnWindows),
+            "If true, allows the user to mount 'tasks' volume read-only on Windows OS",
+            new RuntimeKnobSource("VSTS_SETUP_ALLOW_MOUNT_TASKS_READONLY"),
+            new EnvironmentKnobSource("VSTS_SETUP_ALLOW_MOUNT_TASKS_READONLY"),
+            new BuiltInDefaultKnobSource("false"));
```

Comment:
@sdobrodeev Let's default this to true. The knob will just be a workaround if there are unforeseen consequences.

Reply:
@mjroghelia done! Thanks for the review.